Mathlib/Data/List/DropRight.lean

```diff
@@ -247,4 +247,22 @@ theorem rtakeWhile_idempotent : rtakeWhile p (rtakeWhile p l) = rtakeWhile p l :
   rtakeWhile_eq_self_iff.mpr fun _ => mem_rtakeWhile_imp
 #align list.rtake_while_idempotent List.rtakeWhile_idempotent
 
+
+lemma rdrop_add {l : List α} (i j : ℕ) :
+    l.rdrop (i + j) = (l.rdrop i).rdrop j := by
+  simp_rw [List.rdrop_eq_reverse_drop_reverse, reverse_reverse,
+           drop_drop, Nat.add_comm]
```

Comment:
Is there a reason for LHS/RHS inconsistency between `List.drop_drop` and `List.rdrop_add`?

Reply:
I didn't see one - flipped.